sphinxcontrib/spelling/builder.py

```diff
@@ -61,15 +61,35 @@ def init(self):
             f.append(filters.ImportableModuleFilter)
         f.extend(self.config.spelling_filters)
 
-        project_words = os.path.join(self.srcdir,
-                                     self.config.spelling_word_list_filename)
-        self.info('Looking for custom word list in {}'.format(
-            project_words))
+        if not os.path.isdir(self.outdir):
+            os.mkdir(self.outdir)
+
+        #
+        # In case the user has multiple word lists, we combine them into one
+        # large list that we pass on to the checker.
+        #
+        word_list = self.config.spelling_word_list_filename
+        # If we have a list, the combined list is the first list plus all words
+        # from the other lists. Otherwise, word_list is assumed to just be a
+        # string.
+        if isinstance(word_list, list):
+            combined_word_list = os.path.join(self.outdir,
+                                              'spelling_wordlist.txt')
```

Comment:
This will result in the wordlist being published, won't it? Can we write to a temporary directory instead?

Reply:
Ah yes it would. I didn't think of the outdir as something to publish, but that makes sense. I'll try to change it to mkdtemp or friends instead.